src/types/webhook.ts

```diff
@@ -2,6 +2,7 @@ import type { NextFunction, Request } from 'express';
 
 export type WebhookHandlerOptions = {
   parse: false;
+  disableEncryption?: boolean;
```

Comment:
It feels kind of weird to have this as an option when you haven't enabled encryption... like if you just updated the dependencies and you show the intellisense for other keys and you see this, I feel like you'd think encryption was on already.

[Looks like you're already pulling from the `config` to figure out what encryption to use](https://github.com/Pantheon-Labs/Jiter-Node/pull/27/files#diff-d5ece93cf2632bf0816e81afa8648d532fdc4e9dac90b8774e1997c418b982e7R55), why not just always pull from that?

If we want to allow certain webhooks to have/not have encryption, I feel like this would be the place to do something like `encryptionOverride` or something that forcefully enables/disables encryption regardless of what the `config` has. Thoughts?

Reply:
Yeah this was to disable it for certain events, there's also one on the `createEvent` one. I could put the jsdoc so it explains what it does or change it's name, but it's supposed to be for a one time disable. The example I was thinking of was calling a webhook on another service that may not have the keys so you wouldn't want that encrypted, and then since I had it on the `createEvent` function I figured it should also be here. I'm totally open to renaming/moving/removing it though.